src/runtime/crt/aot_executor/aot_executor.c

```diff
@@ -37,29 +37,26 @@ tvm_crt_error_t tvm_runtime_run(const tvm_model_t* model, void** inputs, void**
   TVMValue tvm_values[model->num_input_tensors + model->num_output_tensors];  // NOLINT
   int32_t tvm_typeids[model->num_input_tensors + model->num_output_tensors];  // NOLINT
 
-  for (int i = 0; i < model->num_input_tensors; i++) {
-    tensors[i] = (DLTensor){
-        .device = fake_device,
-        .data = inputs[i],
-        .shape = &fake_shape,
-        .ndim = fake_dims,
-        .byte_offset = 0,
-        .strides = NULL,
-    };
+  for (uint32_t i = 0; i < model->num_input_tensors; i++) {
```

Comment:
```suggestion
  for (size_t i = 0; i < model->num_input_tensors; i++) {
```

Reply:
I think this is better but we will have to change the type of `model->num_input_tensors` as well.